src/System.Private.ServiceModel/src/System/ServiceModel/Dispatcher/StreamFormatter.cs

```diff
@@ -348,5 +360,81 @@ public void ReleaseStream(Stream stream)
                 //Noop
             }
         }
+
+        internal class StreamFormatterHelper
```

Comment:
Recommend adding comments whether this code was duplicated from somewhere else and why (i.e. not available in public contract).